src/graphnet/data/dataset/parquet/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""Datasets using parquet backend."""
+# Configuration
+from graphnet.utilities.imports import has_torch_package
+
+if has_torch_package():
+    import torch.multiprocessing
+    from .parquet_dataset import ParquetDataset
+
+    torch.multiprocessing.set_sharing_strategy("file_system")
+
+del has_torch_package
```

Comment:
Is there a reason for deleting the `has_torch_package` here but keeping it in `src/graphnet/data/dataset/sqlite/__init__.py`?

Reply:
I actually don't know. I did not add this line; the difference you point out here is present in the main branch currently also. I have added the `del` statement to the sqlite part now too.